src/components/TransactionHistory/TransactionsGroupByDate.tsx

```diff
@@ -22,13 +22,12 @@ export const TransactionsGroupByDate: React.FC<TransactionsGroupByDateProps> = m
   ({ txIds, useCompactMode = false }) => {
     const { setNode, entry } = useResizeObserver()
     const transactions = useAppSelector(state => selectTxDateByIds(state, txIds))
-    const borderTopColor = useColorModeValue('gray.100', 'gray.750')
     const txRows = useMemo(() => {
       const groups: TransactionGroup[] = []
       for (let index = 0; index < transactions.length; index++) {
         const transaction = transactions[index]
         const transactionDate = dayjs(transaction.date * 1000)
-          .startOf('day')
+          .startOf('month')
```

Comment:
This is a little weird now, as we just show the month and the time. Was this an intentional change?

<img width="1482" alt="Screenshot 2023-08-14 at 8 39 26 am" src="https://github.com/shapeshift/web/assets/97164662/2c8a8637-26bc-4157-a2f9-6b598141b2e6">

Note all of my scam NFTs 😕

Reply:
Yah I like grouping my month more than by day, but didn't update the format to include the full date on each transaction. But I can just leave this out for another future PR is not that big of a deal.